controller/internal/webhook/systemVerifier.go

```
@@ -0,0 +1,62 @@
+	if parsedURL.Port() == "" {
+		parsedURL.Host = fmt.Sprintf("%s:443", parsedURL.Hostname())
+	}
+
+	// Dial the server using the correct host and port
+	conn, err := tls.Dial("tcp", parsedURL.Host, &tls.Config{
+		RootCAs: r.roots,
+	})
+	if err != nil {
+		return fmt.Errorf("failed to establish TLS connection: %v", err)
+	}
+	defer conn.Close()
```

golangci-lint (1.62.2)

Error return value of `conn.Close` is not checked

(errcheck)